app/controllers/jobs_controller.rb

```diff
@@ -39,6 +39,9 @@ def show_errored_job
   end
 
   def show_submitted_job
-    response.status = :accepted
+    respond_to do |format|
+      format.html { response.status = :accepted }
+      format.json { render json: { status: :accepted } }
```

Comment:
Would it be better to use `status: :submitted` instead to make it consistent with other jobs?

Reply:
actually we may just want to use accepted rather than submitted